baxter_gazebo/CMakeLists.txt

```diff
@@ -1,31 +1,27 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(baxter_gazebo)
 
-SET(CMAKE_CXX_FLAGS "-std=c++11")
```

Comment:
This is automatically set by the line ``+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${GAZEBO_CXX_FLAGS}")``